src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -671,12 +671,14 @@ void TwitchMessageBuilder::appendUsername()
 
     switch (usernameDisplayMode.getValue())
     {
-        case UsernameDisplayMode::Username: {
+        case UsernameDisplayMode::Username:
```

Comment:
Update your clang

Reply:
everytime i do sth, the formatting is scuffed, what are you doing with it all the time

Reply:
Formatting doesn't change that often, downloading clang 9.0+ should fix it